Core/System.cpp

```diff
@@ -707,18 +707,42 @@ Path GetSysDirectory(PSPDirectories directoryType) {
 	}
 }
 
-void CreateSysDirectories() {
+bool CreateSysDirectories() {
+#if PPSSPP_PLATFORM(ANDROID)
+	const bool createNoMedia = true;
+#else
+	const bool createNoMedia = false;
+#endif
+
+	Path pspDir = GetSysDirectory(DIRECTORY_PSP);
+	INFO_LOG(IO, "Creating '%s' and subdirs:", pspDir.c_str());
+	File::CreateDir(pspDir);
+	if (!File::Exists(pspDir)) {
+		INFO_LOG(IO, "Not a workable memstick directory. Giving up");
+		return false;
+	}
+
 	// Create the default directories that a real PSP creates. Good for homebrew so they can
 	// expect a standard environment. Skipping THEME though, that's pointless.
-	File::CreateDir(GetSysDirectory(DIRECTORY_PSP));
-	File::CreateDir(GetSysDirectory(DIRECTORY_PSP) / "COMMON");
```

Comment:
Supposedly, per #9333, COMMON was for homebrew or something?  I've never actually seen it used, just noting.

-[Unknown]

Reply:
Oh yeah, didn't even noticed I lost it. As you say, quite obscure, not sure if worth adding back..